apps/sms/views/conversation/test/unit/conversation_test.js

```diff
@@ -6385,7 +6400,9 @@ suite('conversation.js >', function() {
         Promise.resolve(message)
       );
 
-      ConversationView.handleActivity({ messageId: message.id }).then(() => {
+      ConversationView.afterEnterComposer(
```

Comment:
we shouod _really_ use properly beforeEnter/afterEnter with transition args. But these tests will be rewritten in bug 1162030 anyway so I can do that...